tests/consensus_tests/create_collection_and_check.py

```diff
@@ -15,6 +15,8 @@
         "distance": "Dot",
         "shard_number": 6
     })
+print(r)
```

Comment:
I think those 2 println are an oversight :)

Reply:
Yes, they were, though as we use this function for testing, I am inclined to leave them here still :smile: . What do you think?